front-end/src/pages/CreateAccount.js

```diff
@@ -0,0 +1,167 @@
+import React, { useState } from 'react';
+import { Grid } from '@material-ui/core';
+import { ExpandLess } from '@material-ui/icons';
+import { Link } from '@material-ui/core';
+import TextInput from '../components/TextInput';
+import PasswordInput from '../components/PasswordInput';
+import {
+	GreenTextTypography,
+	GreenSmallTypographyCreate,
+	TitleTextTypography,
+	TitleText1Typography,
+	CreateButton,
+	CreateLock,
+	LogoCreateUserIcon,
+	LogoCreateTitle,
+	backgroundStyle,
+	ValidatiionPasswordString
+} from '../components/Themes';
+import '../assets/styles/login.scss'
+
+export default function Login() {
+	const [username, setUsername] = useState("");
+	const [password, setPassword] = useState("");
+	const [firstname, setFirstname] = useState("");
+	const [lastname, setLastname] = useState("");
+	const [repassword, setRepassword] = useState("");
+	const [passwordValation, setPasswordValidation] = useState(false)
+
+	const handleSubmit = (event) => {
```

Comment:
Validate

- Username, first name and last name are not blank
- password is more than eight characters
- both passwords are the same

Show red validation text for ALL invalid form fields
;